docs/metrics-3.6.md

```diff
@@ -0,0 +1,81 @@
+# Metrics upgrade notes
+
+Metrics have changed between 3.5 and 3.6.
+
+You can now retrieve metrics using the OpenTelemetry Protocol using the [OpenTelemetry collector](https://opentelemetry.io/docs/collector/), and this is the recommended mechanism.
+
+These notes explain the differences in using the prometheus `/metrics` endpoint to scrape metrics for a minimal effort upgrade. It is not recommended you follow this guide blindly, the new metrics have been introduced because they add value, and so they should be worth collecting and using.
+
+## TLS
+
+The prometheus `/metrics` endpoint defaults to TLS on.
+
+To disable this set `.metricsConfig.secure` to `false`.
+
+## New metrics
+
+The following are new metrics:
+
+* `build_info`
+* `total_count`
+* `pods_total_count`
+* `controller_build_info`
+* `cronworkflows_triggered_total`
+* `workflowtemplate_triggered_total`
+* `workflowtemplate_runtime`
+* `k8s_request_duration`
+* `queue_duration`
+* `queue_longest_running`
+* `queue_retries`
+* `queue_unfinished_work`
+* `pod_pending`
+
+and can be disabled with
+
+```yaml
+metricsConfig:
+  options:
+    build_info:
+      disable: true
+    total_count:
+      disable: true
+    pods_total_count:
+      disable: true
+    controller_build_info:
+      disable: true
+    cronworkflows_triggered_total:
+      disable: true
+    workflowtemplate_triggered_total:
+      disable: true
+    workflowtemplate_runtime:
+      disable: true
+    k8s_request_duration:
+      disable: true
+    queue_duration:
+      disable: true
+    queue_longest_running:
+      disable: true
+    queue_retries:
+      disable: true}
+    queue_unfinished_work:
+      disable: true
+    pod_pending:
+      disable: true
+```
+
+## Renamed metrics
+
+If you are using these metrics in your recording rules, dashboards or alerts you will need to use their new name after the upgrade:
```

Comment:
```suggestion
If you are using these metrics in your recording rules, dashboards, or alerts, you will need to update their names after the upgrade:
```

missing commas

Also I really think this section and the next one should be above "New Metrics" -- they are breaking changes and so are most notable, while the rest are additions. It should be in order of importance / highest impact to existing users